app/models/comment.rb

```diff
@@ -1,4 +1,10 @@
 class Comment < ApplicationRecord
-  belongs_to :user
-  belongs_to :post
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  after_save :update_comment_counter
+
+  def update_comment_counter
+    post.increment!(:Comments_Counter)
+  end
```

Comment:
### OPTIONAL
You can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_comment_counter method every time a new post has been saved to the database, then make the update_comment_counter method private
```ruby
after_save :update_comment_counter

  private

  def update_comment_counter
    post.increment!(:Comments_Counter)
  end
```